mantidproject/mantid
# Merge Changes From Rob and Richard for ISIS SANS

Fixes trac issue [#11360](http://trac.mantidproject.org/mantid/ticket/11360).

**To tester**

These are changes given to us by Rob Dalgliesh and Richard Heenan, which I'm merging for them almost completely "as is".  I have:
- given the code a manual inspection to see that it is sensible;
- fixed a couple of issues that cropped up with the fact that they were using slightly outdated versions of files; and
- fixed broken system tests.

I am submitting this pull request so that I may provide them both with a build which they can use to give the changes a final "once-over" before the cycle starts this week.

For testing purposes, I'd recommend waiting until we hear back from the scientists that the changes are working as they intended, then simply do a code review.

Rob has said that he would like to provide us with system test data / reduction scripts so that we may have LARMOR covered with adequate test coverage.  Having said that, they are very keen on getting something they can use into the nightly builds ASAP, so we will have to do this in a separate ticket at a later date.
